twingly_search/client.py

```diff
@@ -52,34 +62,41 @@ def query(self):
         """
         return Query(self)
 
-    def execute_query(self, query):
+    def execute_query(self, q):
```

Comment:
@roback it will!

I've forgotten about simple type checking as I mostly work with Java :-)

I'll try to update sources today, but do not guarantee that will have enough time.

Thanks,
Yuri.